release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -132,6 +132,13 @@ With this update, you can configure the SR-IOV Network Operator to drain nodes i
 
 For further information, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#configure-sr-iov-operator-parallel-nodes_configuring-sriov-device[Configuring parallel node draining during SR-IOV network policy updates].
 
+[id="ocp-4-16-networking-user-managed-lb-baremetal"]
```

Comment:
I think we should not only mentioned on baremetal? it's also supported vsphere/nutanix/openstack

Reply:
Thanks, @zhaozhanqi . Updated. I mentioned "on-site" infrastructure to avoid a collison with "on-premise" in a style guide. 
![Screenshot from 2024-05-27 14-43-43](https://github.com/openshift/openshift-docs/assets/57954076/617c77d7-f659-4d7f-8e08-bbfeb5a63014)

Reply:
Thanks, @cybertron . I'm bringing this to the style council tomorrow to clarify things. I agree that in the context of OCP that "on-premise" is a regular term and the style guidance needs to be clarified.

Reply:
I'm sticking with "on-premise" now as the guideline can be revisited next week.